hazelcast/src/main/java/com/hazelcast/map/impl/LocalMapStatsProvider.java

```diff
@@ -131,6 +135,7 @@ public LocalMapStatsImpl createLocalMapStats(String mapName) {
             LocalMapOnDemandCalculatedStats onDemand = ((LocalMapOnDemandCalculatedStats) entry.getValue());
             addNearCacheStats(mapName, existingStats, onDemand);
             addIndexStats(mapName, existingStats);
+            addStructureStats(mapName, onDemand);
```

Comment:
do we need this same line at `createLocalMapStats(mapName)`

Reply:
I think it does that: `createLocalMapStats(mapName)` -> `updateMapOnDemandStats` -> `addStructureStats`.

Reply:
I see it now. Thanks